### PR TITLE
Update AnimatedVisualSources

### PR DESCRIPTION
**Pull request by StephenLPeters**
New lottie animations which remove the opacity issues.

### CODE REVIEW

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml [hunk 1]
```diff
@@ -336,34 +336,6 @@
                                         <Setter Target="ContentPresenter.Foreground" Value="{ThemeResource NavigationViewButtonForegroundDisabled}" />
                                     </VisualState.Setters>
                                 </VisualState>
-                                <VisualState x:Name="Checked">
```
Comment (ranjeshj) [inline dev/NavigationView/NavigationView_rs1_themeresources.xaml:339]: Are these changes intentional ?

File: dev/dll/Microsoft.UI.Xaml.vcxproj.filters [hunk 2]
```diff
@@ -294,6 +294,78 @@
     <Page Include="@(PageRequiringCustomCompilation)" />
     <Page Include="@(PageRequiringCustomCompilation)" />
     <Page Include="@(PageRequiringCustomCompilation)" />
+    <Page Include="@(PageRequiringCustomCompilation)" />
+    <Page Include="@(PageRequiringCustomCompilation)" />
```
Comment (jevansaks) [inline dev/dll/Microsoft.UI.Xaml.vcxproj.filters:298]: Arg this is happening again. Please revert.

**Review by jevansaks — APPROVED** [review]
:shipit:

File: dev/NavigationView/NavigationView_rs1_themeresources.xaml (hunk 1 above)
Comment (StephenLPeters) [reply dev/NavigationView/NavigationView_rs1_themeresources.xaml:339]: Yes, this is the style for the hamburger button which is a button, not a toggle button.  These states have never been used.

File: dev/dll/Microsoft.UI.Xaml.vcxproj.filters (hunk 2 above)
Comment (ranjeshj) [reply dev/dll/Microsoft.UI.Xaml.vcxproj.filters:298]: Hmm. thought we fixed this a while back. Lets follow up in a separate change.
